test/lib/dwolla_test.exs

```diff
@@ -27,80 +24,94 @@ defmodule DwollaTest do
     end
 
     test "make_request_with_token/3 raises when root_uri is missing" do
+      Dwolla.Mock
+      |> expect(:request, 1, fn :get, _, _, _, _ ->
+        raise Dwolla.HttpClient.MissingRootUriError
+      end)
+
       Application.put_env(:dwolla, :root_uri, nil)
```

Comment:
Technically, this isn't doing anything since Mox is catching the call. Should we be using Mox to catch this in the first place? Wouldn't this test fail properly if the Mox expect was removed and the `:root_uri` is removed from the environment?

Reply:
Good catch. This actually isnt necessary anymore after I refactored the http_client usage into a separate module HttpClient. Previously, this exception wasn't getting triggered with the Mox function since process_url function that calls the require_root_uri function wasn't called. But with the refactor to have HttpClient, it now is. 

However, Ill be removing this test altogether given above comment about this exception being unnecessary